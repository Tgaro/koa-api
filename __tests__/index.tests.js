const request = require('supertest-as-promised')
const app = require('../config/app')
const userModel = require('../app/models/user').model

describe("routes: index", () => {
	it('Should get one user from mongodb', async () => {
		const res = await request(app.listen())
			.get('/')

		expect(res.status).toBe(200)
		expect(res.header['content-type']).toBe('application/json; charset=utf-8')
		expect(parseInt(res.header['content-length'])).not.toBeLessThan(0)
		expect(res.header['content-length']).not.toBe('0')
	})
})