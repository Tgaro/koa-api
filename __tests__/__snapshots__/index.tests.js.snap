// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes: index Should get one user from mongodb 1`] = `
Object {
  "header": Object {
    "connection": "close",
    "content-length": "1163",
    "content-type": "application/json; charset=utf-8",
    "date": "Fri, 30 Nov 2018 13:12:17 GMT",
  },
  "req": Object {
    "data": undefined,
    "headers": Object {
      "user-agent": "node-superagent/3.8.3",
    },
    "method": "GET",
    "url": "http://127.0.0.1:50427/",
  },
  "status": 200,
  "text": "[{\\"token\\":\\"123456\\",\\"user\\":\\"Tiago\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"2988390\\",\\"user\\":\\"Tiago Aro\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0},{\\"token\\":\\"4988\\",\\"user\\":\\"Rafael\\",\\"pass\\":\\"ABCD\\",\\"__v\\":0}]",
}
`;
